tests/integration/test_integration_v05.py

```diff
@@ -0,0 +1,50 @@
+import sys
+
+import pytest
+
+import ddtrace
+from ddtrace.internal.encoding import MsgpackEncoderV05
+import ddtrace.internal.writer
+import ddtrace.tracer
+from tests.integration.test_integration import *  # noqa
```

Comment:
maybe a comment here on what is going on?

Reply:
as icky as it may feel I think it's worth either completely copy-pasting these tests or attempting to parametrize them. `import *` can cause really subtle weird issues which might eat up someone's day trying to debug

Reply:
I think parametrisation is definitely the way to go in this case, but I'm regarding this test source as temporary until we have the code for the activation of the new encoder in place; then we could think of the best way to test the encoders while reusing as much as possible.

Reply:
I still think a comment here would help so people understand where the tests are coming from

Reply:
I've added the comment on L1. Unfortunately I wasn't able to add it on this particular line because of Black 😭 .